sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/hibernate/reactive/AbstractDatabaseHibernateReactiveIT.java

```diff
@@ -297,5 +301,21 @@ public void newLineInQuery() {
         assertThat(author.body().asString(), containsString("Slovník"));
     }
 
+    @Tag("https://github.com/quarkusio/quarkus/issues/47033")
+    @Test
+    public void nativeBuildCompilationWithValidList() {
```

Comment:
Thinking about this and I don't think it's needed. The coverage in my POV is that this module/quarkus app compile in native. Similar to https://github.com/quarkus-qe/quarkus-test-suite/pull/1962/files where it's just tested compilation. But maybe I'm mistaken

Reply:
Yeah, you are right, I think I could keep the test to a healthy post-compilation flag, but I will remove the test eventually